### PR TITLE
Updates SSL cert proc for Operator docs

### PR DESCRIPTION
**Pull request by stevsmit**
https://issues.redhat.com/browse/PROJQUAY-7623

### CODE REVIEW

File: modules/creating-custom-ssl-certs-config-bundle.adoc [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+After creating your custom SSL/TLS certificates, you can create a custom `configBundleSecret` resource for {productname-ocp}, which allows you to upload `ssl.cert` and `ssl.key` files.
+
+.Prerequisites
+
+* You have base64 decoded the original config bundle into a `config.yaml` file.
+* You have generated custom SSL certificates and keys.
+
+.Procedure
+
+. If not already set, update your `config.yaml` file to include the `PREFERRED_URL_SCHEME: https`, `EXTERNAL_TLS_TERMINATION: false`, and `SERVER_HOSTNAME` fields:
```
Comment (SeanZhao-redhat) [inline modules/creating-custom-ssl-certs-config-bundle.adoc:14]: `PREFERRED_URL_SCHEME: https`, `EXTERNAL_TLS_TERMINATION: false` is not necessary, it only must route:unmanaged

File: modules/creating-custom-ssl-certs-config-bundle.adoc [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+.. Create the resource by entering the following command:
++
+[source,terminal]
+----
+$ oc -n <namespace> create secret generic custom-ssl-config-bundle-secret \
+  --from-file=config.yaml=<config.yaml> \ <1>
+  --from-file=ssl.cert=<ssl.cert> \ <2>
+  --from-file=extra_ca_cert=<ca.cert> \ <3>
+  --from-file=ssl.key=<ssl.key> \ <4>
+  --dry-run=client -o yaml > custom-ssl-config-bundle-secret.yaml
```
Comment (ibazulic) [inline modules/creating-custom-ssl-certs-config-bundle.adoc:32]: The `extra_ca_cert` must have its own name:

```
...
--from-file=extra_ca_cert_{NAME-OF-CERT}.crt=ca-certificate-bundle.crt
...
```

File: modules/creating-custom-ssl-certs-config-bundle.adoc [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+----
+<1> Where `<config.yaml>` is your `base64 decoded` `config.yaml` file.
+<2> Where `<ssl.cert>` is your `ssl.cert` file.
+<3> Optional. The `--from-file=extra_ca_cert=<ca.cert>` field allows {productname} to recognize custom Certificate Authority (CA) files. If you are using LDAP, OIDC, or another service that uses custom CAs, you must add them via the `extra_ca_cert` path.
+<4> Where `<ssl.key>` is your `ssl.key` file.
```
Comment (ibazulic) [inline modules/creating-custom-ssl-certs-config-bundle.adoc:37]: Same as above.

File: modules/creating-custom-ssl-certs-config-bundle.adoc [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: v1
+data:
+  config.yaml: QUxMT1dfUFVMTFNfV0lUSE9VVF9TVFJJQ1RfTE9HR0lORzogZmFsc2UKQVVUSEVOVElDQVRJT05fVFlQRTogRGF0YWJhc2UKREVGQVVMVF9UQUdfRVhQSVJBVElPTjogMncKRElTVFJJQlVURURfU1R...
+  ssl.cert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVYakNDQTBhZ0F3SUJBZ0lVTUFBRk1YVWlWVHNoMGxNTWI3U1l0eFV5eTJjd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2dZZ3hDekFKQmdOVkJBWVR...
+  extra_ca_cert:LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVYakNDQTBhZ0F3SUJBZ0lVTUFBRk1YVWlWVHNoMGxNTWI3U1l0eFV5eTJjd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2dZZ3hDe...
```
Comment (ibazulic) [inline modules/creating-custom-ssl-certs-config-bundle.adoc:54]: Same as above. The key name must be in the form `extra_ca_cert_{NAME-OF-CERT}.crt`.

**Pull request merged**